src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -356,6 +356,40 @@ protected boolean putToImpl(String body, String serviceUrl) {
         return sendToImpl(body, serviceUrl, getContentType(), "PUT");
     }
 
+    /**
+     * Sends a file by HTTP PUT body to service endpoint.
+     * @param fileName fileName to post
+     * @param serviceUrl service endpoint to send body to.
+     * @return true if call could be made and response did not indicate error.
+     */
+    public boolean putFileTo(String fileName, String serviceUrl) {
+        return putFileToImpl(fileName, serviceUrl);
+    }
+
+
+    protected boolean putFileToImpl(String fileName, String serviceUrl) {
+        boolean result;
+        resetResponse();
+        String url = getUrl(serviceUrl);
+
+        String filePath = getFilePathFromWikiUrl(fileName);
+        File file = new File(filePath);
+        if (!file.exists()) {
+            throw new StopTestException(false, "File " + filePath + " not found.");
+        }
+
+        try {
+            response.setRequest(fileName);
+            storeLastCall("PUT_FILE", serviceUrl);
```

Comment:
Please also update `repeatLastCall()` to support PUT_FILE

Reply:
Done!